DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/EditDelegateController.cs

```diff
@@ -75,7 +75,8 @@ public IActionResult Index(EditDelegateFormData formData, int delegateId)
                 return ReturnToEditDetailsViewWithErrors(formData, delegateId, centreId);
             }
 
-            if (!userService.NewEmailAddressIsValid(formData.Email!, null, delegateId, User.GetUserId()!.Value))
+            // TODO HEEDLS-887 Fix this so that it is bassing the correct User ID, that of the delegate not the logged in admin

```

Comment:
@GeorgeBlackwell Is this being done in your ticket? We can talk about it tomorrow.

Reply:
This is really https://softwiretech.atlassian.net/browse/HEEDLS-951 I don't see why @GeorgeBlackwell would be changing this on his ticket? I'll mark it with the correct todo number